composer.json

```diff
@@ -48,7 +48,8 @@
         "mongodb/mongodb": "^1.0",
         "predis/predis": "^1.0",
         "squizlabs/php_codesniffer": "~2.0",
-        "vlucas/phpdotenv": "^2.4.0"
+        "vlucas/phpdotenv": "^2.4.0",
+        "symfony/process": "^3.3"
```

Comment:
Other Symfony dependencies are supported down to version 2.7, I think that this dependency should have the same version constraint.